src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -64,7 +64,7 @@ const useGroupSetting = ({route}) => {
         channel_name: groupName,
         channel_image: urlImage
       };
-      if (urlImage) {
+      if (urlImage && !isValidUrl(urlImage)) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the check for `urlImage && !isValidUrl(urlImage)` is good, it might be better to provide feedback to the user when the URL is invalid. Currently, if the URL is invalid, the upload process is simply bypassed without any indication to the user as to why.

```diff
- if (urlImage && !isValidUrl(urlImage)) {
+ if (urlImage) {
+   if (!isValidUrl(urlImage)) {
+     SimpleToast.show('Invalid image URL');
+     return;
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
I think this is necesarry because we have two condition and nested condition should not necesarry